src/sagas/fetch/index.js

```diff
@@ -18,4 +18,24 @@ export function* fetchSaga(
   return null
 }
 
+export function* fetchAnythingSaga(action: Object): Generator<any, any, any> {
+  try {
+    yield put(actions.clearError(action.name))
+    yield put(actions.sendingRequest(action.name, true))
+    const response = yield call(get, action.query, action.args)
+    if (action.callback) {
+      const callbacks = action.callback(response, action.args)
+      while (callbacks.length) {
```

Comment:
Seems a bit more idiomatic to just do a map, `callbacks.map(callback => yield put(callback))`